tests/pkg_tiny-asn1/main.c

```diff
@@ -203,6 +203,6 @@ int main(void)
     free(asn1_objects);
 
     /* Everything worked fine */
-    printf("Decoding finished succesfully\n");
+    printf("Decoding finished successfully\n");
```

Comment:
Apparently this also needs to be fixed in the test script :woman_facepalming:

Reply:
fixed